Artifacts/windows-vsts-build-agent/vsts-agent-install.ps1

```diff
@@ -247,6 +247,78 @@ function Install-Agent
 
         if ($Config.RunAsAutoLogon)
         {
+            # Create a PS session for the user to trigger the creation of the registry entries required for autologon

```

Comment:
A good part of this is the same logic as already reviewed earlier. Please consider those comments as applicable here as well.

Reply:
done